### ISSUE TITLE
Linux npm install fail on most distribution

### ISSUE THREAD

**Issue by altagir** [lightvector/arimaa-server#130]
Python2 is getting quite old, it is not even supported anymore on most linux systems
Including pi raspberry

so npm install fails with 

```
npm ERR! path /opt/arimaa-server/node_modules/node-sass
npm ERR! gyp verb check python checking for Python executable "python2" in the PATH
npm ERR! gyp verb `which` failed Error: not found: python2

```
Also many warnings of deprecated libs, any plan to update ?

**Comment by lightvector** [comment]
Yes, nobody has touched this repo in years, so at this point it's just a static archive. If you are working on it and want to update it and become the new de-facto owner/maintainer, you can make your own fork (subject to the license).
